pr_2/javascript/random_samples/insecure_cors_policy.js

```
@@ -0,0 +1,28 @@
+function insecureCorsPolicyNoncompliant() {
+    app.post('/users', function (req, res) {
+        const origin = req.query.origin
+        // Noncompliant: the Access-Control-Allow-Origin header is set to user-controlled any domain.
+        res.set(200, {'Access-Control-Allow-Origin': origin })
+    })
+}
+// {/fact}
+
+
+// {fact rule=insecure-cors-policy@v1.0 defects=0}
+var express = require("express")
```

Biome (2.1.2)

[error] Shouldn't redeclare 'express'. Consider to delete it or rename it.

'express' is defined here:


(lint/suspicious/noRedeclare)